<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerManager</class>
 <widget class="QDialog" name="LayerManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Layer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="createButton">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QTableWidget" name="layers">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Layer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tags</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="addTagButton">
         <property name="text">
          <string>Add Tag</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="deleteTagButton">
         <property name="text">
          <string>Delete Tag</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tags</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tags">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">7</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">9</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">10</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">11</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">12</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">13</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">14</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">15</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">16</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">17</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">18</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">19</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">20</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">21</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">22</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">23</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">24</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">25</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">26</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">27</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">28</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">29</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">30</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string notr="true">31</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Tag</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>layers</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>addTagButton</tabstop>
  <tabstop>deleteTagButton</tabstop>
  <tabstop>tags</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LayerManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LayerManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
